client/tailscale/keys.go

```diff
@@ -13,6 +13,10 @@ import (
 	"time"
 )
 
+type GetKeysResponse struct {
```

Comment:
Could you please make this unexported (`getKeysResponse`) and move it closer to the `Keys` function where it's used (perhaps even inline like the `var keys` was before)?

Reply:
Happy to!  My intention was to keep it consistent with [devices.go](https://github.com/tailscale/tailscale/blob/6edf357b96b28ee1be659a70232c0135b2ffedfd/client/tailscale/devices.go#L21-L23)

Reply:
✅